docs/source/format/Flight.rst

```diff
@@ -360,6 +367,52 @@ string, so the obvious candidates are not compatible.  The chosen
 representation can be parsed by both implementations, as well as Go's
 ``net/url`` and Python's ``urllib.parse``.
 
+.. _flight-extended-uris:
+
+Extended Location URIs
+----------------------
+
+In addition to alternative transports, a server may also return
+URIs that reference an external service or object storage location.
+This can be useful in cases where intermediate data is cached as
+Apache Parquet files on cloud storage or is otherwise accessible
+via an HTTP service. In these scenarios, it is more efficient to be
+able to provide a URI where the client may simply download the data
+directly, rather than requiring a Flight service to read it back into
+memory and serve it from a ``DoGet`` request.
+
+To avoid the complexities of Flight Clients having to implement support
+for multiple different cloud storage vendors (e.g. AWS S3, Google Cloud),
```

Comment:
This is slightly idealistic.  Every vendor could have a different signing process and, since only HTTP URIs are shared, you are putting the burden on the client to figure out how to sign the requests.

Reply:
the purpose here is that there's *no* burden on the client, it just makes a GET request (or uses pre-negotiated auth)